.github/workflows/release-drafter-fa.yml

```diff
@@ -0,0 +1,110 @@
+name: Schedule a release and draft notes for function app
```

Comment:
Basically the same comments as with the previous file. Including the newline at the end.